src/getDefaultSequencerInboxMaxTimeVariation.ts

```diff
@@ -0,0 +1,29 @@
+import { PublicClient } from 'viem';
+
+import { ParentChainId, validateParentChain } from './types/ParentChain';
+
+export type SequencerInboxMaxTimeVariation = {
+  delayBlocks: bigint;
+  futureBlocks: bigint;
+  delaySeconds: bigint;
+  futureSeconds: bigint;
+};
+
+export function getDefaultSequencerInboxMaxTimeVariation(
+  parentChainIdOrPublicClient: ParentChainId | PublicClient,
+): SequencerInboxMaxTimeVariation {
+  validateParentChain(parentChainIdOrPublicClient);
```

Comment:
right now it's unused but will be used in a future PR when we add support for Base